cpp/unreal_plugins/UrdfBot/Source/UrdfParser.h

```diff
@@ -0,0 +1,142 @@
+//
+// Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+//
+
+#pragma once
+
+#include <Eigen/Dense>
+#include <map>
+#include <vector>
+
+#include <CoreMinimal.h>
+
+class FXmlNode;
+
+struct UrdfJointDesc;
+struct UrdfLinkDesc;
+struct UrdfMaterialDesc;
+
+enum UrdfGeometryType
+{
+    INVALID_GEOMETRY = 0,
```

Comment:
Our convention for enums is as follows:

```
enum UrdfGeometryType
{
    Invalid  = 0,
    Box      = 1,
    Sphere   = 2,
    Cylinder = 3,
    Mesh     = 4,
};

Reply:
fixed

Reply:
having `Invalid` in both `UrdfGeometryType` and `UrdfJointType` cause redefinition. Use `enum class` for both to resolve this issue.

Reply:
Ah you're right. And that's what our existing enums do too. They're defined as `enum class`.